src/client.js

```diff
@@ -183,6 +198,14 @@ export default class StitchClient {
     )
       .then(response => response.json());
   }
+
+  /**
+  * @return {Promise} whether or not the current client is authenticated
+  */
+  isAuthenticated() {
+    return this.auth.authedId().then((id) => id != null && id != undefined);
```

Comment:
[minor] This promise can reject. If it's suppose to return a bool, we should probably satisfy the `reject` case on `then`.
[test] Test for when it rejects